docs/coordinates/skycoord.rst

```diff
@@ -1127,7 +1128,7 @@ The example below illustrates the floating point issue using the exact
 equality comparison, where we do a roundtrip transformation
 FK4 => ICRS => FK4 and then compare::
 
-  >>> sc1 = SkyCoord(1*u.deg, 2*u.deg, frame='fk4')
+  >>> sc1 = SkyCoord(1*u.deg, 2*u.deg, representation_type="unitspherical", frame='fk4')
```

Comment:
Otherwise, the comparison fails because `sc1.icrs.fk4` and `sc1` have different representation types. Similarly to what happens with the doctest above, not sure if we should try to avoid this.

Reply:
Scratch my comment above, this is still failing.